CONTRIBUTING.md

```diff
@@ -37,12 +37,14 @@ The best bug reports include a detailed way to predictably reproduce the issue,
-* **Weekly status updates** from individual community members are posted as GitHub issues labeled [Meeting Notes](https://github.com/ampproject/amphtml/issues?q=label%3A%22Meeting+Notes%22).  If you have a weekly status update related to your work on AMP that you'd like to share with the community please add it as a comment on the relevant Meeting Notes issue.
-* **Weekly design reviews** are held as video conferences via Google Hangouts on Wednesdays at [1pm Pacific](https://www.google.com/?#q=1pm+pacific+in+local+time).  Design reviews are used to discuss/refine engineering designs after an initial draft of the design has been created and shared with the community.
+* **Weekly status updates** from individual community members are posted as GitHub issues labeled [Type: Weekly Status](https://github.com/ampproject/amphtml/issues?q=label%3A%22Type%3A+Weekly+Status%22).  If you have a weekly status update related to your work on AMP that you'd like to share with the community please add it as a comment on the relevant Weekly Status issue.
+* **Weekly design reviews** are held as video conferences via Google Hangouts on Wednesdays at [1pm Pacific](https://www.google.com/?#q=1pm+pacific+in+local+time).  Design reviews are used to discuss/refine engineering designs after an initial draft of the design has been created and shared with the community.  The design reviews are meant as an *optional* venue for concentrated feedback and discussion.  **Going through this design review is *not* required to make a contribution to AMP.**
   * We use GitHub issues labeled [Type: Design Review](https://github.com/ampproject/amphtml/issues?q=label%3A%22Type%3A+Design+Review%22) to track design reviews.  The Design Review issue for a given week will have a link to the design docs being discussed that week as well as a link to the Hangout.
   * When you attend a design review please read through the design docs before the review starts.
   * If you have an engineering design you would like to discuss at a design review:
-    * Document your design in a shared Google Document open to public comments.  A short design doc is fine as long as it covers your design in sufficient detail to allow for a review by other members of the community.  Take a look at [Design docs - A design doc](https://medium.com/@cramforce/design-docs-a-design-doc-a152f4484c6b) for tips on putting together a good design doc.
+    * Document your design in a shared Google Document open to public comments.  A short design doc is fine as long as it covers your design in sufficient detail to allow for a review by other members of the community.  Take a look at [Design docs - A design doc](https://medium.com/@cramforce/design-docs-a-design-doc-a152f4484c6b) for tips on putting together a good design doc.  Some example design docs:
```

Comment:
Done.  I reworded the first sentence to include "software" and left the rest as design docs to avoid redundancy.